chain/market/fundmanager.go

```diff
@@ -428,31 +491,27 @@ func (a *fundedAddress) processWithdrawals(reserved abi.TokenAmount) (pr *proces
 
 	// Check if there is anything to withdraw
 	if allowedAmt.Equals(abi.NewTokenAmount(0)) {
-		// Mark allowed requests as complete
+		// Mark allowed requests as cancelled
 		for _, req := range allowed {
```

Comment:
totally just curious: what is the scenario where allowedAmt == 0 and len(allowed) != 0 (I guess if you withdraw 0?)